pkg/render/guardian_test.go

```diff
@@ -49,6 +49,7 @@ var _ = Describe("Rendering tests", func() {
 	var resources []client.Object
 
 	createGuardianConfig := func(i operatorv1.InstallationSpec, addr string, openshift bool) *render.GuardianConfiguration {
+		i.Variant = operatorv1.TigeraSecureEnterprise
```

Comment:
Seems odd that we are arbitrarily settings this to enterprise here - why not in the place that passes the Installation to this function?

Reply:
This is to keep these current tests running since it was expected that all of them are for EE.

Reply:
Yeah I know that but why is it jammed in here and not wherever the rest of the `InstallationSpec` that is given is generated? What if we want to run any of these tests against a different Variant now that Guardian is deployed for Calico as well?